src/main/java/org/fcrepo/auth/webac/URIConstants.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.auth.webac;
+
+import java.net.URI;
+
+/**
+ * URIs used by the WebAC module.
+ *
+ * @author Peter Eichman
+ * @since Aug 25, 2015
+ * @see <a href="http://www.w3.org/wiki/WebAccessControl/Vocabulary">
+ *      http://www.w3.org/wiki/WebAccessControl/Vocabulary</a>
+ * @see <a href="http://www.w3.org/ns/auth/acl">http://www.w3.org/ns/auth/acl</a>
+ */
+public abstract class URIConstants {
```

Comment:
I've always just defined a private constructor to prevent instantiation of the class. That's what I've seen others do, but this works for me, too. @awoods do you have an opinion on this?